.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly be noticed that the .gitignore file listing `test.md` file and `node_modules` directory.